test/parallel/test-strace-openat-openssl.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const common = require('../common');
+const { spawn, spawnSync } = require('node:child_process');
+const { createInterface } = require('node:readline');
+const assert = require('node:assert');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+if (!common.isLinux)
+  common.skip('linux only');
+if (spawnSync('strace').error !== undefined) {
```

Comment:
Not sure if that's the best way to check it. cc: @nodejs/build

Reply:
Perhaps check 'strace -V' exits with `.status === 0`